From: Jason Rhinelander <jason@imaginary.ca>
Date: Thu, 29 Oct 2015 16:35:34 -0400
Subject: Eigen 3.2.2 fix: don't specify size of null vector

This results in a compilation error on jessie, and isn't needed (the
default constructor does what we want anyway).
---
 src/eris/belief/BayesianLinearRestricted.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/eris/belief/BayesianLinearRestricted.cpp b/src/eris/belief/BayesianLinearRestricted.cpp
index 01db10a..fb082a5 100644
--- a/src/eris/belief/BayesianLinearRestricted.cpp
+++ b/src/eris/belief/BayesianLinearRestricted.cpp
@@ -168,7 +168,7 @@ void BayesianLinearRestricted::gibbsInitialize(const Ref<const VectorXd> &initia
 
     if (not gibbs_r_Rbeta_) gibbs_r_Rbeta_.reset(
             restrict_size_ == 0
-            ? new VectorXd(0)
+            ? new VectorXd()
             : new VectorXd(r() - R() * beta_));
     const auto &r_minus_Rbeta_ = *gibbs_r_Rbeta_;
 
@@ -230,7 +230,7 @@ const VectorXd& BayesianLinearRestricted::drawGibbs() {
 
     if (not gibbs_r_Rbeta_) gibbs_r_Rbeta_.reset(
             restrict_size_ == 0
-            ? new VectorXd(0)
+            ? new VectorXd()
             : new VectorXd(r() - R() * beta_));
     const auto &r_minus_Rbeta_ = *gibbs_r_Rbeta_;
 
